sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suites.scala

```diff
@@ -1189,6 +1189,7 @@ abstract class HiveThriftServer2TestBase extends SparkFunSuite with BeforeAndAft
   protected val startScript = "../../sbin/start-thriftserver.sh".split("/").mkString(File.separator)
   protected val stopScript = "../../sbin/stop-thriftserver.sh".split("/").mkString(File.separator)
 
+  val localhost = Utils.localHostNameForURI()
```

Comment:
Shall we use `Utils.localCanonicalHostName` as https://github.com/apache/spark/pull/37377 suggested?

Reply:
Oh, did you hit any failure, @cloud-fan ? We can if we need. IIUC, `KafkaTestUtils` needs it previously due to MiniKDC `principal` string.

Reply:
yea I found it when running this test in our internal test infra. Opened https://github.com/apache/spark/pull/37656